shangridocs-webapp/src/main/webapp/META-INF/context.xml

```diff
@@ -17,11 +17,14 @@ the License.
 -->
 <Context path="/shangridocs">
         
-    <Parameter name="shangridocs.homepage"
-        value="org.shangridocs.HomePage"/>  
+    <Parameter name="shangridocs.homepage.default"
+        value="org.shangridocs.HomePage"/>
+          
+    <Parameter name="shangridocs.homepage.v2"
```

Comment:
i don't understand these params, it's kind of confusing. Why not just make shangridocs.homepage reference the page class, and set the V2page to be the default. We don't need to introduce shangridocs.homepage.v2.